src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/NuGetUI.cs

```diff
@@ -58,10 +61,12 @@ public sealed class NuGetUI : INuGetUI
             ICommonOperations commonOperations,
             NuGetUIProjectContext projectContext,
             INuGetUILogger logger,
-            NuGetUIContext uiContext)
+            INuGetUIContext uiContext,
+            IPackageManagerControlViewModel packageManagerControlViewModel)
```

Comment:
A separate call to set `PackageManagerControl=Control` still happens elsewhere. Introducing an injected ViewModel here allowed me to test this PR, but also can serve as a stepping stone to making the PM UI use more MVVM.